src/opa.js

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by an Apache2
 // license that can be found in the LICENSE file.
 const builtIns = require("./builtins/index");
-const utf8 = require("utf8");
+const util = require("util");
```

Comment:
Sorry, is this meant to be used in the browser? I'm getting an error because `TextDecoder is not a constructor` and as far as I can see, `util` is a native Node library and I don't seem to be able to use it within a React application. Thanks!

Reply:
Have you seen https://github.com/open-policy-agent/npm-opa-wasm/pull/110?

It's not in a release yet but it should address this